src/core/utils/EventNormalizer.ts

```diff
@@ -152,24 +155,16 @@ export class EventNormalizer {
      * @private
      * @returns {CompiledEvent}
      */
-    _compileEvents(): CompiledEvent {
-        if (this._compiledEvent) {
-            return this._compiledEvent;
+    _addSimplifiedEvent(ev: CompiledEvent): void {
```

Comment:
At this point, I wonder what is the point of CompiledEvent. Couldn't we just use Event here ?

Reply:
I wanted to see with you, in fact I asked myself the question. Yes, I'm using native event.